packages/expo-store-review/ios/StoreReviewModule.swift

```diff
@@ -0,0 +1,26 @@
+import ExpoModulesCore
+import StoreKit
+
+public class StoreReviewModule: Module {
+  public func definition() -> ModuleDefinition {
+    Name("ExpoStoreReview")
+
+    AsyncFunction("isAvailableAsync") { () -> Bool in
+      return true
+    }
+
+    AsyncFunction("requestReview") { (promise: Promise) -> Void in
+      if #available(iOS 15, *) {
+        guard let currentScene = UIApplication.shared.connectedScenes.first as? UIWindowScene else {
+          return promise.reject(MissingCurrentWindowSceneException())
```

Comment:
Even though this is an async function, errors thrown synchronously also get caught and the promise is then automatically rejected, so you can simplify that

```suggestion
          throw MissingCurrentWindowSceneException()
```